src/features/utils/aggregate-base.js

```diff
@@ -0,0 +1,7 @@
+import { FeatureBase } from "./feature-base";
+
+export class AggregateBase extends FeatureBase {
```

Comment:
Minor gripe about this serving no purpose.

Reply:
More of a pattern to follow for future modifications if necessary